src/RuleSet.php

```diff
@@ -255,6 +254,7 @@ final class RuleSet implements RuleSetInterface
             'phpdoc_var_annotation_correct_order' => true,
             'protected_to_private' => true,
             'return_assignment' => true,
+            'echo_tag_syntax' => true,
```

Comment:
Should be moved after `compact_nullable_typehint` for alphabetical order.

Reply:
Done: dbefb78ccb2a48ea59f31abc94cc0d2358b4718e